cpp-tour/cpp-tour
# Design / Compiler Improvements

## What does it do?
Several improvements:

- CSS / Design improvements
- Show warnings & errors from compiler output
- Correctly handle compiler errors

## How can this be tested?

- Put a `#warning "foo"` above main, to see compiler warnings
- Put a `#error "foo"` above main, to see an error

## Preview

![capture](https://user-images.githubusercontent.com/396475/33810449-e854c0d6-de04-11e7-951f-ef5e5779702a.PNG)
